src/DOM.js

```diff
@@ -2072,6 +2072,9 @@ class HTMLIFrameElement extends HTMLSrcableElement {
   }
   sendKeyDown(key, modifiers) {
     this.browser && this.browser.sendKeyDown(key, modifiers);
+	if(key === 13){
```

Comment:
It's helpful for these PRs to either set up the editor to match style of these files, or do a conversion after.

Otherwise things quickly become unreadable on Github.

Reply:
yes i will do that ,